packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/telephony-line-phone-order.service.js

```diff
@@ -0,0 +1,32 @@
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+import groupBy from 'lodash/groupBy';
+import head from 'lodash/head';
+import map from 'lodash/map';
+
+export default function filterContact(contacts) {
+  return filter(
+    map(
+      groupBy(contacts, (contact) => {
+        // group contact to detect contact that are the same
+        const contactCopy = {
+          lastName: contact.lastName,
+          firstName: contact.firstName,
+        };
+        if (contact.address) {
+          contactCopy.address = {
+            country: contact.address.country,
+            line1: contact.address.line1,
+            zip: contact.address.zip,
+            city: contact.address.city,
+          };
+        }
+        return JSON.stringify(contactCopy);
```

Comment:
This function does too much, it filters and it convert to string, maybe you could do this outside the function ?

Reply:
Perhaps...

Reply:
It was an existing code, just moved.